sympy/assumptions/ask.py

```diff
@@ -1263,7 +1263,7 @@ def ask(proposition, assumptions=True, context=global_assumptions):
         Relations in assumptions are not implemented (yet), so the following
         will not give a meaningful result.
 
-        >>> ask(Q.positive(x), Q.is_true(x > 0)) # doctest: +SKIP
+        >>> ask(Q.positive(x), Q.is_true(x > 0))
```

Comment:
This is kind of a weird doctest. The answer here should be True. It will give True in https://github.com/sympy/sympy/pull/17392.

Reply:
I was also thinking about that. Should I leave it as is (as that will cause a test failure to be fixed in #17392) or should I skip it here and change the answer to the correct one and hope that someone remembers to enable it later?

Reply:
I was thinking about another thing. I'll leave it as is. Returning None is not incorrect at least.